packages/template-retail-react-app/app/hooks/use-wish-list.js

```diff
@@ -31,7 +31,7 @@ export const useWishList = ({listId = ''} = {}) => {
         }
     )
     const wishLists = productLists?.data?.filter((list) => list.type === 'wish_list') || []
-    const currentWishlist = wishLists?.data?.find((list) => list.id === listId) || wishLists[0]
+    const currentWishlist = wishLists.find((list) => list.id === listId) || wishLists[0]
```

Comment:
We do we use `wishLists[0]` as a fallback, when we know that it doesn't have the correct ID?

Reply:
Ah right, that's a good point.. we wanted to use the first wishlist _only_ when the caller does not pass in the list id. Let me fix that.

Reply:
TBH, in the context of retail react app, you never need to pass in list id.

Reply:
I see. We can either mimic the existing logic (of using a name and a type): https://github.com/SalesforceCommerceCloud/pwa-kit/blob/0ef7c22630ef0fa2e62ecf932bff07003ff01866/packages/template-retail-react-app/app/hooks/use-wishlist.js#L12-L13 

Or use the "first wishlist that has type wish_list". Whichever we decide to do, let's do it in a separate PR.